.github/pull_request_template.md

```diff
@@ -23,3 +23,16 @@ Thank you for opening a pull request. Please provide:
 - [ ] Public functions and types are documented
 - [ ] Standard formatting is applied to Go code
 - [ ] Is this a new API? Added a new file that begins with `//go:build ceph_preview`
+- [ ] Record API changes: run `make api-update`
```

Comment:
To better align with the checklist mode used in already existing items how about we rephrase the above into following:

"- [ ] Ran `make api-update` to record API changes."

Reply:
Great suggestion. I tweaked it just a little to include the word "new" to additionally  imply the two bullets are related.